### DISCUSSION TITLE
part-ii/2017-3/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#181] [Question comments]
Part II 2017 Algebraic Geometry: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2017-3/

**Comment by silkca** [comment]
The first three parts are all simple bookwork.

The final part is easy both from an algebraic and geometric perspective - the implication of this question seems to be that the algebraic side is easier to handle, but I don't believe that is even true.
